doc/en/user/source/extensions/geofence-server/gui.rst

```diff
@@ -37,3 +37,20 @@ When Access type ACCESS is selected as well as a specific layer, it becomes poss
 
 .. figure:: images/layerdetails.png
    :align: center
+
+
+Layer groups
+^^^^^^^^^^^^
+Layer groups are also supported. If no workspace has been specified, the layer dropdown will show global layer groups, while if a workspace is selected the workspace's layergroup will be showed together with the layers.
+
+The read and write filters text areas as well as the style palette in the layer details tab are disabled when the layer group is being configured.
+
+If an allowed area WKT is specified it will be applied to the contained layers.
+
+When a layer contained in a layer group is directly accessed in the context of a WMS request, the following rules apply:
```

Comment:
Documentation should also explain what's happening when multiple groups contain the same layer. The code seems to have different approaches for spatial filtering and alphanumeric filtering, not sure they are implemented consistenty either. Please clarify, make sure the code follows it consistently, and add tests to cover it.

Reply:
fixed